src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -1023,21 +1021,20 @@ export default {
 
 	.filters {
 		position: absolute;
-		top: calc(var(--default-grid-baseline) * 2);
-		right: calc(var(--default-grid-baseline) * 2 + var(--default-clickable-area));
+		top: 0;
+		right: calc(var(--default-grid-baseline) + var(--default-clickable-area));
 	}
 
 	.actions {
 		position: absolute;
-		top: calc(var(--default-grid-baseline) * 2);
-		right: calc(var(--default-grid-baseline) * 1);
+		top: 0;
+		right: 0;
 	}
 }
 
 .invitation-button {
-	padding: 0 10px;
-	overflow-y: scroll; // align total width with list items
-	margin-bottom: 4px;
+	padding-inline: calc(var(--default-grid-baseline) * 2);
+	margin-block: var(--default-grid-baseline);
 
 	:deep(.app-navigation-entry-link) {
 		padding-left: 10px;
```

Comment:
```suggestion
```

to be inline with list items

Reply:
![image](https://github.com/user-attachments/assets/fc7fb60a-154c-427f-bdbf-2f180e4f7d74)